README.md

```diff
@@ -35,3 +35,4 @@ Unzipped the full parquet data will be approximately 70Gb. Each (compressed) chu
 ### Glossary
 
 - TLD means Top-level Domain. You can read up more about it [here.](https://en.wikipedia.org/wiki/Top-level_domain) 
+- [Web crawler](https://en.wikipedia.org/wiki/Web_crawler)- It is an Internet bot that systematically browses the World Wide Web, typically for the purpose of Web indexing (web spidering).
```

Comment:
This definition of a web crawler is not really appropriate for the type of crawl we've done in this repo. Specifically we have not done the typical thing of web indexing so mentioning that here could add confusion if not contextualized. I don't see a need to mention web indexing.

Reply:
I have made the changes. Hope, this is fine.